.cirrus.yml

```diff
@@ -49,7 +49,7 @@ task:
   use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
   name: build-ipas
   osx_instance:
-    image: high-sierra-xcode-9.4.1
+    image: catalina-flutter
```

Comment:
Shouldn't we use `catalina-xcode-12.2` so we're controlling the version of Xcode rather than having it randomly change out from under us. (I know that's currently happening, but only because this has become so stale.)

Alternately should we go all the way to `big-sur-xcode-12.4`?

Reply:
Done.